rhel/SPECS/grafana.spec

```diff
@@ -105,7 +105,7 @@ exit 0
 
 %changelog
```

Comment:
Please add a new record to this section with ticket details and your email

Reply:
Done